cypress/config/dev.json

```diff
@@ -1,7 +1,7 @@
 {
   "baseUrl": "https://dev-madie.hcqis.org",
   "screenshotsFolder": "mochawesome-report/assets",
-  "numTestsKeptInMemory": 0,
+  "numTestsKeptInMemory": 20,
```

Comment:
why are you changing this? We had put this to 0 for the memory issues we were having in Jenkins a while back.

Reply:
oh sorry I changed it in my local for debugging purpose & pushed it by accident. Will revert it